CHANGELOG.md

```diff
@@ -13,6 +13,11 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - The metric portion of the OpenCensus bridge (`go.opentelemetry.io/otel/bridge/opencensus`) has been reintroduced. (#3192)
 - The OpenCensus bridge example (`go.opentelemetry.io/otel/example/opencensus`) has been reintroduced. (#3206)
 
+### Changed
+
+- `span.SetStatus` has been updated to comply with the OpenTelemetry specification.
+  Calls that lower the status are now noops. (#3214)
```

Comment:
```suggestion
- `span.SetStatus` has been updated such that calls that lower the status are now no-ops. (#3214)
```

The existing behavior was compliant, since the spec said attempts to lower the status value should be ignored, not that they must be ignored.